src/streaming/models/VideoModel.js

```diff
@@ -151,6 +152,22 @@ MediaPlayer.models.VideoModel = function () {
         setElement: function (value) {
             element = value;
         },
+        
+        getTTMLRenderingDiv: function () {
+            return TTMLRenderingDiv;
+        },
+        
+        setTTMLRenderingDiv: function (div) {
```

Comment:
@KozhinM @TobbeMobiTV  - Michael,  I told Tobbe that maybe we can add API to mediaplayer to set external rendering div.. I then told him to store div reference in video model as it seems related to element.  Any issue with this.  Not sure if it overloads this object too much.  I think it works but wanted more opinion.